openshift/system/Dockerfile

```diff
@@ -57,6 +57,8 @@ RUN export ${BUNDLER_ENV} >/dev/null \
     && chmod g+w -vfR log tmp public/assets db/sphinx \
     && umask 0002 \ 
     && cd /opt/system \
+    && npm set progress false \
+    && npm install \
```

Comment:
This can be done sooner. Just just bundler just adds Gemfile and runs this can be just add package-lock.json and run npm and be own layer.

Reply:
So, I tried to move this up but then `npm` is not found. See [#38939](https://circleci.com/gh/3scale/porta/38939) and [#38871](https://circleci.com/gh/3scale/porta/38871) for instance.